Formula/qt5.rb

```diff
@@ -48,13 +48,23 @@ class Qt5 < Formula
     sha256 "2cf77b820f46f0c404284882b4a4a97bf005b680062842cdc53e107a821deeda"
   end
 
+  # Fix build error under El Capitan/Sierra (10.11/10.12), under these version
+  # bluetooth class moved from IOBluetooth to CoreBluetooth, so anycode still
+  # using IOBluetooth will fail to build.
+  # This patch also fix the missing symbol in cups backend of qtwebengine
+  patch do
+    url "https://gist.githubusercontent.com/steve3d/b246f2eb35e5b9f89d1dcaa3470da59d/raw/75ba606df18d2a4901ff12b5ddd78c41a1711509/qt5-bluetooth-cups-mac-10.1x.patch"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!